apps/demos/Demos/Menu/Scrolling/Angular/app/app.component.css

```diff
@@ -0,0 +1,38 @@
+html,
+body {
+  width: 100%;
+  height: 100%;
+}
```

Comment:
do we need these styles? isn't it supposed to work the same without it?

Reply:
Without those, Angular TestCafe fails. I applied how it was styled initially on jQuery which was approved last week. It's possible to not have these but we will remove it for all frameworks.